scripts/export-m3u8s.js

```diff
@@ -0,0 +1,82 @@
+/* eslint no-console: 0 */
+
+let fs = require('fs');
+let path = require('path');
+
+let basePath = path.resolve(__dirname, '..');
+let testDataDir = path.join(basePath, 'test');
+let manifestDir = path.join(basePath, 'test', 'fixtures', 'm3u8');
+let manifestFilepath = path.join(testDataDir, 'test-manifests.js');
+let expectedFilepath = path.join(testDataDir, 'test-expected.js');
+
+let build = function() {
+  let manifests = 'export default {\n';
```

Comment:
Since this is part of the build process, I have deliberately made it synchronous for simplicity. We don't need web-scale throughput for this :)